freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -57,20 +60,38 @@ def handle(self, event):
         db_event = Event.get_or_create_from_event(db.session, event)
         self.set_context(db_event)
 
-        auto_build_images_list = self._get_auto_rebuild_image_list(event)
-        rebuild_images = self._filter_images_with_higher_rpm_nvr(
-            event, auto_build_images_list
+        self.errata = Errata()
+        self.advisory_module_nvrs = self.errata.get_cve_affected_build_nvrs(
+            event.advisory.errata_id, True
         )
-        if not auto_build_images_list or not rebuild_images:
-            msg = "There is no image can be rebuilt. " f"message_id: {event.msg_id}"
+
+        auto_build_image_modules_mapping = self._get_auto_rebuild_image_mapping()
+        if auto_build_image_modules_mapping:
+            images = list(auto_build_image_modules_mapping.keys())
+            rebuild_images = self._filter_images_with_higher_rpm_nvr(images)
+        if not auto_build_image_modules_mapping or not rebuild_images:
+            msg = (
+                "Images are not enabled for auto rebuild. "
+                if not auto_build_image_modules_mapping
+                else "No images are impacted by the advisory. "
+            )
+            msg = f"{msg} message_id: {event.msg_id}"
             db_event.transition(EventState.SKIPPED, msg)
             db.session.commit()
             self.log_info(msg)
             return []
 
-        images_nvr = ",".join([rebuild_image.nvr for rebuild_image in rebuild_images])
+        images_nvr = ",".join([rebuild_image.nvr() for rebuild_image in rebuild_images])
```

Comment:
I might be wrong, but this change looks like a typo and seems there are no tests to catch this - i.e. missing test where `_filter_images_with_higher_rpm_nvr()` returns non-empty list.

Reply:
Good catch. yeah, It is a typo. I updated the test_record_builds a bit to cover this.